dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/ExceptionDeserializingErrorDecoder.java

```diff
@@ -74,22 +74,20 @@ final class ExceptionDeserializingErrorDecoder {
             JSON_ENCODING.deserializer(new TypeMarker<>() {});
     private static final Deserializer<SerializableError> SERIALIZABLE_ERROR_DESERIALIZER =
             JSON_ENCODING.deserializer(new TypeMarker<>() {});
+
     private final Map<String, DeserializerExceptionPair<?, ?>> errorNameToExceptionDeserializerMap;
+    private final boolean expectRichErrors;
```

Comment:
```suggestion
    private final boolean expectJsonErrors;
```

v nit: because I suggested we check that the format is explicitly `JSON_FORMAT` when populating this variable in `ConjureBodySerde`